exercises/custom-set/example.ml

```diff
@@ -1,3 +1,5 @@
+open Core.Std
+
```

Comment:
Core requires a massive buy in to the Core ecosystem. We have many small packages in OCaml that provide what you most likely were looking for, like string manipulation and whatnot.